README.md

```diff
@@ -133,6 +133,10 @@ The following merge options are supported:
   repositories. Set this option to `true` to automatically delete branches
   after they have been merged.
 
+- `MERGE_DELETE_PROTECT_BRANCHES`: A comma-separated list of branches that will not
```

Comment:
MERGE_DELETE_PROTECT_BRANCHES to me sounds a bit like "also delete protected branches" or so, which doesn't make sense. How about MERGE_DELETE_BRANCH_FILTER or so?

Reply:
Changed to `MERGE_DELETE_BRANCH_FILTER`.